docs/concepts-replication.md

```diff
@@ -39,24 +40,26 @@ Synchronous geo-replication provides the highest availability and also guarantee
 
 ## Replication patterns
 
-Pulsar provides a great degree of flexibility for customizing your replication strategy. You can set up different replication patterns to serve your replication strategy for an application between multiple data centers.
+Pulsar provides a great degree of flexibility for customizing your replication strategy. You can set up different replication patterns to serve your replication strategy for an application between multiple data centers. 
+
+Pulsar supports the following three replication patterns:
```

Comment:
```suggestion
Pulsar supports the following replication patterns:
```
Avoid specifying specific NO. if there are more replication patterns to be supported in the future.